catalog/admin/admin-poller-service/src/test/java/org/codice/ddf/catalog/admin/poller/AdminPollerTest.java

```diff
@@ -183,11 +183,11 @@ protected AdminSourceHelper getHelper() {
                 // Mock out the metatypes
```

Comment:
I looked into this, all the changes are inside a private helper class without public interfaces, and the current unit tests are mocking out everything that my changes do. I'm not sure how to Unit test without either making the helper's filter generation method public or extracting the helper class (which I'm pretty sure only exists to mock output we'd have to get from the OSGi environment for easier unit testing).
We might be able to add it to an integration test.